ci/owncloud/upload_to_owncloud.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+from os import path
+from getpass import getuser
+import netrc
+from webdav3.client import Client
+from uuid import uuid4
+import json
+from opts import debug, message, opts
+
+default_hostname = "https://ftp.essensium.com/owncloud/remote.php/dav/files/"
+
+
+def get_credentials(options_path=None, force_netrc=True):
+    """
+    get_credentials(options_path=None, force_netrc=True)
+
+    Gets the user's default credentials from either an external options file or ~/.netrc
+
+    Parameters
+    ----------
+    options_path : str, optional
+                path to the options file
+
+    force_netrc : bool, optional
+                flag indicating whether to skip parsing the options file and instead directly
+                load the credentials from ~/.netrc
+    Returns
+    -------
+    dictionary with 3 key/value pairs,
+    keys are: {webdav_login", "webdav_password","webdav_hostname")
+    """
+    if not force_netrc and path.exists(options_path):
+        with open(options_path) as options_file:
+            options = json.load(options_file)
+            return options
```

Comment:
Even when reading from netrc, the rest of the options should be taken from options_path.

Also, there should be a default for it.